spec/features/user_past_order_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+feature "Past Orders" do
+  let(:user) { User.create(fullname: "Jeff Casmir", email: "demo+jeff@jumpstartlab.com", display_name: "j3", password: "password", role: 0)}
+  let(:order) { Order.create(user_id: user.id, status: "Completed", total: 1000)}
+  let(:item) { Item.create(title: "Two Torpedo Tacos", description: "Two crispy chicken tacos.", price: 500, image:"torpedo_tacos.png")}
```

Comment:
Line is too long. [136/80]<br>Space missing after colon.<br>Space missing inside }.